### PR TITLE
Protiflio desktop version

### PR DESCRIPTION
**Pull request by beteland123**
Used media queries to display correctly on desktop screens.
Added Grid and Flexbox In order to lay out the elements on the page.
Added CSS style in order to make it the same as the Figima template

### CODE REVIEW

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @beteland123, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 


- Descriptive Pull Request Title and Summary. ✔️  
- Media Query Perfectly used ✔️  
- Flexbox and Grid in order to layout the elements ✔️
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

- [ ] [Optional] I could not highlight in as an inline comment because your README file is not part of the files changed. Please kindly work on the following section in your readme file to make it look more professional than what you already have.

1.  kindly add the name of your project `i.e My-Portfolio` in the project_name header instead of having it as `project_name`
2.  kindly add the correct link of your social media handle to the author section
3.  kindly remove the link you put in the live demo section since you didn't deploy the project yet you can remove it or write `not available` as the content

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -803,3 +838,904 @@
...
+  .TO-BE-HIDDEN,
+  #ruby {
+    display: flex;
+    flex-wrap: nowrap;
+    height: 2em;
+    margin-right: 0;
+  }
+
+  *html {
+    margin: 0;
```
[the remark this answers is not in the document]
Comment (beteland123) [reply style.css]: I tried to make it the same as the Figma design precisely the elements under card class frame2 but it didn't work. can you please tell me what the problem is even if I make the same font size with the Figma it doesn't display as it needs to be

File: index.html [hunk 2]
```diff
@@ -16,16 +16,23 @@
...
+            </ul>
+            <img src="img/1.jpg" class="icon" >
+        <nav>
     </header>
     
     
     <section class="headline">
         <div class="text">
-            <h1 class="primery-text">I'm Betel. Glad to see you!</h1>
+            <h1 class="primery-text">I'm Betel . Glad to see you!</h1>
```
Comment (Qoosim) [inline index.html:32]: - [ ] Great job implementing the changes from the previous reviewer. However, there're few things you still need to work on before your project could be approved. It is required to stick to the figma design as close as possible. Take a closer look at the main title on the figma design. `I'm Betel` should be on one line while `Glad to see you!` should be on the next line after the first line. But your title does not follow this design closely. Kindly fix that.

### Your Design
your-main-title

### Figma Design
temp-main-title

File: index.html [hunk 3]
```diff
@@ -92,36 +99,43 @@ <h3>2015</h3>
...
-                    <div class="client">
-                        <h3 >
+                    <div class="client" id="ctop">
+                        <h3 class="DESKTOP-HIDDEN" >
                         CANOPY
                         </h3>
+                        <h3 class="TO-BE-HIDDEN">
+                            FACEBOOK
+                        </h3>
                     </div>
```
Comment (Qoosim) [inline index.html:118]: - [ ] The project cards on your page do not fall to the center of the page as it is on the figma design. Kindly make them to be at the center of your page.
- [ ] Also, the padding of the language lists of each project should be as wide as the content in it. In your case, javascript and css have the same padding which make the css to look distorted in a way. Kindly attend to that too and fix. Please, take a look :point_down: 

### Your Design
your-project-card

### Figma Design
temp-project-card

File: index.html [hunk 4]
```diff
@@ -214,6 +252,7 @@ <h3>2015</h3>
     </section>
    
 </section>
+
 <section class="about-section">
     <section class="top">
         <h2 class="prim-text">About my self</h2>
```
Comment (Qoosim) [inline index.html:258]: - [ ] There's little thing you need to work on your about myself section. The information part of this section covers 40% of the width while the language part covers the remaining 60% as we have it on the figma design. Kindly fix that too.

### Your Design
your-about-me

### Figma Design
temp-about-me

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @beteland123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @beteland123 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**